common/djangoapps/student/tests/test_models.py

```diff
@@ -103,3 +103,16 @@ def test_users_enrolled_in_all(self):
             CourseEnrollment.objects.users_enrolled_in(self.course.id, include_inactive=True)
         )
         self.assertListEqual([self.user, self.user_2], all_enrolled_users)
+
+    def test_users_enrolled_in_without_unauth_or_lti(self):
```

Comment:
If we reword this method to remove mention of unauth (and really, LTI for that matter) and just refer to this as `users_enrolled_without_fake_email` we can submit this upstream.